dashboard/src/utils/helper.js

```diff
@@ -44,7 +44,7 @@ export const validateEmail = (email) => {
  * @param {Object} args - value for each templated parameter
  * @return {string} - formatted URI
  */
-export const expandUriTemplate = (endpoints, name, args) => {
+export const uriTemplate = (endpoints, name, args) => {
```

Comment:
It would be handy to set the default for `args` to an empty object, so that the caller can omit it when it has nothing to provide.
```javascript
export const uriTemplate = (endpoints, name, args = {}) => {
```

Reply:
Maybe; there are only a few that don't, and it didn't seem worthwhile at the time.